geoapi/utils/lidar.py

```diff
@@ -8,63 +8,64 @@
 from typing import List
 
 
-def _transform_to_geojson(epsg, point: tuple) -> tuple:
+def _transform_to_geojson(proj4, point: tuple) -> tuple:
     """
     Transform point to epsg:4326
-    :param epsg: int
+    :param proj4: proj string
     :param point
     :return: point
     """
-    input_projection = Proj(init='EPSG:{}'.format(epsg))
+    input_projection = Proj(proj4)
     geojson_default_projection = Proj(init="epsg:4326")
     x, y = transform(input_projection, geojson_default_projection, point[0], point[1], errcheck=True)
     return x, y
 
 
-class Lidar:
-
-    @staticmethod
-    def getEPSG(filePath: str):
-        """
-        Get EPSG of las file
-        :param filePath
-        :return: int
-        """
+def getProj4(filePath: str):
+    """
+    Get proj4 of las file
+    :param filePath
+    :return: str
+    :raises InvalidCoordinateReferenceSystem
+    """
+    result = subprocess.run([
```

Comment:
I'm still using subprocess as the `pdal info` command/workflow is not available to the python module.